content/en/blog/_posts/2022-08-14-pod-has-network-condition.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
+date: 2022-05-27
```

Comment:
Please update this field _and_ rename the file, to match the assigned publication date: 2022-09-14.

Reply:
done